src/main/java/teammates/ui/webapi/GetFeedbackSessionAction.java

```diff
@@ -71,4 +69,39 @@ public JsonResult execute() {
 
         return new JsonResult(response);
     }
+
+    private FeedbackSessionData getFeedbackSessionDataWithFilteredDeadlines(Intent intent) {
+        String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
+        String feedbackSessionName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        FeedbackSessionAttributes feedbackSessionAttributes = getNonNullFeedbackSession(feedbackSessionName, courseId);
```

Comment:
Any reasons why this is moved inside?

Reply:
Those variables were not necessary in the information hiding in `execute`, so I just chose to move it inside to simplify the method signature rather than passing it in.

Reply:
Ok. I don't have a problem with this.